src/panel.ts

```diff
@@ -28,8 +32,31 @@ export class CommentPanel extends Panel {
     node.setAttribute('contentEditable', 'true');
     node.classList.add('jc-CommentInput');
     const inputWidget = (this._inputWidget = new Widget({ node }));
-    this.addWidget(inputWidget);
 
+    let filePath: string; 
+    options.labShell.currentChanged.connect((_, args) => {
+      if (args.newValue instanceof DocumentWidget){
+        const docWidget = args.newValue as DocumentWidget;
+        filePath = docWidget.context.path;
+        const aware = this.awareness;
+        console.log(filePath)
+        const panelHeader: Widget = new PanelHeader({ awareness: aware, filename: filePath});
+        // const panelHeader: Widget = new PanelHeader({ awareness: undefined, filename: filePath});
+        this.addWidget(panelHeader);
+        this.addWidget(inputWidget);
```

Comment:
the input widget is getting removed by the `onUpdateRequest` method (that's actually okay--I removed the input area in my last PR)